config/settings/base.py

```diff
@@ -376,8 +318,8 @@ def env(name, default=NoDefaultValue, type_=str):
     raise ImproperlyConfigured("Missing environment variable: SFDX_HUB_KEY.")
 
 # CCI expects these env vars to be set to refresh org oauth tokens
-environ["SFDX_CLIENT_ID"] = SFDX_CLIENT_ID
-environ["SFDX_HUB_KEY"] = SFDX_HUB_KEY
+os_environ["SFDX_CLIENT_ID"] = SFDX_CLIENT_ID
+os_environ["SFDX_HUB_KEY"] = SFDX_HUB_KEY
```

Comment:
@jerivas I wasn't 100% sure about this, but I don't see a way to add an env var using django-environ.